be/src/storage/rowset_update_state.cpp

```diff
@@ -103,6 +109,129 @@ Status RowsetUpdateState::_do_load(Rowset* rowset) {
     return Status::OK();
 }
 
+struct RowidSortEntry {
+    uint32_t rowid;
+    uint32_t idx;
+    RowidSortEntry(uint32_t rowid, uint32_t idx) : rowid(rowid), idx(idx) {}
+    bool operator<(const RowidSortEntry& rhs) const { return rowid < rhs.rowid; }
+};
+
+static void plan_read_by_rssid(const vector<uint64_t>& rowids, size_t* num_default,
```

Comment:
It's better to add some annotation to describe the function. This function is so important.